go-controller/pkg/cluster/master.go

```diff
@@ -280,27 +269,27 @@ func (cluster *OvnClusterController) SetupMaster(masterNodeName string) error {
 	}
 
 	// Create 2 load-balancers for east-west traffic.  One handles UDP and another handles TCP.
-	k8sClusterLbTCP, stderr, err := util.RunOVNNbctl("--data=bare", "--no-heading", "--columns=_uuid", "find", "load_balancer", "external_ids:k8s-cluster-lb-tcp=yes")
+	cluster.TCPLoadBalancerUUID, stderr, err = util.RunOVNNbctl("--data=bare", "--no-heading", "--columns=_uuid", "find", "load_balancer", "external_ids:k8s-cluster-lb-tcp=yes")
```

Comment:
Would love to see a more parameterized `Find` function, someday.

Reply:
> Would love to see a more parameterized `Find` function, someday.

[Origin's pkg/util/ovs](https://github.com/openshift/origin/tree/master/pkg/util/ovs) might be a good starting point for trying to abstract out the command-line stuff a bit. (I assume "`ovn-nbctl find`" uses the same format as "`ovs-vsctl find`" so the `Find` function there would more-or-less work here.)

Reply:
Yeah, would be nice to have.